common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/WebViewAuthorizationFragment.java

```diff
@@ -120,6 +126,8 @@ public class WebViewAuthorizationFragment extends AuthorizationFragment {
 
     // This is used by LegacyFido2ApiManager to launch a PendingIntent received by the legacy API.
     private ActivityResultLauncher<LegacyFido2ApiObject> mFidoLauncher;
+    // This flag is used to determine if the fragment is waiting for the switch browser resume endpoint.
```

Comment:
Doesn't seem like a flag. This is really an object

Reply:
update comment